### PR TITLE
Feature/milestone5

### PR DESCRIPTION
**Pull request by Dipeshtwis**
What does this PR do:

- Implemented react prop functionality
- Implemented handleClick function
- Added onClick event to all button
- Added style to calculator

### CODE REVIEW

File: src/components/App.js [hunk 1]
```diff
@@ -8,11 +8,36 @@ import calculate from '../logic/calculate';
 
 /* eslint-enable no-unused-vars */
```
Comment (christianotieno) [inline src/components/App.js:7]: ## [OPTIONAL] Stretch Requirement

- [ ] By disabling ESLint rules,  don't you think it rids you the opportunity to write much cleaner code and suppresses the potential warnings you’d be getting, thus missing out on the education that ESLint rules provide? 😟  ([screenshot](https://gitlab.com/microverse/guides/projects/requirements_screenshots/raw/master/images/general/no-ignore-linters.png) from [pathwright](https://microverse.pathwright.com/library/fast-track-curriculum/69047/path/step/54883773/)).

File: src/components/ButtonPanel.js [hunk 2]
```diff
@@ -1,42 +1,47 @@
+import PropTypes from 'prop-types';
 /* eslint-disable no-unused-vars */
```
Comment (christianotieno) [inline src/components/ButtonPanel.js:2]: ## [OPTIONAL] Stretch Requirement

- [ ] By disabling ESLint rules,  don't you think it rids you the opportunity to write much cleaner code and suppresses the potential warnings you’d be getting, thus missing out on the education that ESLint rules provide? 😟  ([screenshot](https://gitlab.com/microverse/guides/projects/requirements_screenshots/raw/master/images/general/no-ignore-linters.png) from [pathwright](https://microverse.pathwright.com/library/fast-track-curriculum/69047/path/step/54883773/)).

File: src/components/Display.js [hunk 3]
```diff
@@ -1,9 +1,10 @@
 import PropTypes from 'prop-types';
 
-const Display = (props = 'O') => <h1>{props.result}</h1>;
+const Display = (props) => <h2 className ='input-value'>{props.result || props.next || '0'}</h2>;
```
Comment (christianotieno) [inline src/components/Display.js:3]: ## Optional 
- [ ] The UI isn't great when it comes to the displaying of what's to get rendered.
While we do have our function store and the next value typed, the characters never get displayed, and the user never gets to see what they keyed; a bad User Experience. Would you mind changing it up a bit (in terms of the display), a user to see the digits (and, hopefully, operations that get typed for great User experience?  This is what we currently have 👇 
calculator

File: src/components/Button.js [hunk 4]
```diff
@@ -1,9 +1,16 @@
 import PropTypes from 'prop-types';
-
-const Button = (props) => <h2>{props.name}</h2>;
+/* eslint-disable no-unused-vars */
```
Comment (christianotieno) [inline src/components/Button.js:2]: ## [OPTIONAL] Stretch Requirement

- [ ] By disabling ESLint rules,  don't you think it rids you the opportunity to write much cleaner code and suppresses the potential warnings you’d be getting, thus missing out on the education that ESLint rules provide? 😟  ([screenshot](https://gitlab.com/microverse/guides/projects/requirements_screenshots/raw/master/images/general/no-ignore-linters.png) from [pathwright](https://microverse.pathwright.com/library/fast-track-curriculum/69047/path/step/54883773/)).

File: src/components/ButtonPanel.js [hunk 5]
```diff
@@ -1,42 +1,47 @@
...
-       </div>
-
-       <div className = "btn-show">
-          {sign5}
-       </div>
+       {displayAllBtns(signArr)}
       </div>
   );
 };
-/* eslint-enable no-unused-vars */
```
Comment (christianotieno) [inline src/components/ButtonPanel.js:41]: ## [OPTIONAL] Stretch Requirement

- [ ] By disabling ESLint rules,  don't you think it rids you the opportunity to write much cleaner code and suppresses the potential warnings you’d be getting, thus missing out on the education that ESLint rules provide? 😟  ([screenshot](https://gitlab.com/microverse/guides/projects/requirements_screenshots/raw/master/images/general/no-ignore-linters.png) from [pathwright](https://microverse.pathwright.com/library/fast-track-curriculum/69047/path/step/54883773/)).

**Review by christianotieno — CHANGES_REQUESTED** [review]
### Changes Required ♻️  
Hello @Dipeshtwis 👋 

The hard work done on this project is commendable and congratulations on reaching this far. Your passion for software development showing with these projects, 🔥 keep it up 💯 

I have come to admire how you have made your repositories to be professional-looking from the strict adherence to the git-flow workflow, to the commits as well. The codebase looks solid and there is a decent use of the convention-based coding when working with React. Good job. 👏 

We can improve on this further, still. Please find below some comments on areas that we can address before proceeding down below, 👇 thank you!
## Required Change 
- [x] We have some commented out code in our main `index.js` module. Kindly remove them. If you need to review a previous version of your code, you can always use git history ([screenshot](https://gitlab.com/microverse/guides/projects/requirements_screenshots/raw/master/images/general/no_inline_comments.png) from [Pathwright](https://microverse.pathwright.com/library/fast-track-curriculum/69047/path/step/66243643/))  
##

### Happy coding 💻 

#### Christian Otieno (TSE)

**Comment by Dipeshtwis** [comment]
The Update project includes:-

- Remove all the unnecessary comment and code

**Review by christianotieno — APPROVED** [review]
### Project Approved 🍾 
Hello @Dipeshtwis 👋 

You have made the required changes needed for approval. Congratulations!
I wish you all the best in the next challenge 🥇 

### Happy coding 💻 
#### Christian Otieno (TSE)

**Pull request merged**